os/kernel/binary_manager/binary_manager_bootparam.c

```diff
@@ -268,22 +269,28 @@ void binary_manager_update_bootparam(int requester_pid, uint8_t type)
 		return;
 	}
 
+	response_msg.result = BINMGR_OK;
+	is_all_updatable = true;
+
 	/* Get current bootparam data and update version */
 	memcpy(&update_bp_data, binary_manager_get_bpdata(), sizeof(binmgr_bpdata_t));
 	update_bp_data.version++;
 
 	if (BM_CHECK_GROUP(type, BINARY_KERNEL)) {
 		/* Update bootparam and Reboot if new kernel binary exists */
 		ret = binary_manager_check_kernel_update(&check_update);
```

Comment:
In `os/kernel/binary_manager/binary_manager_data.c`,
there is no argument in the prototype of `binary_manager_check_kernel_update`.
Please remove the param if not needed.